<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchHDRDialog</class>
 <widget class="QDialog" name="BatchHDRDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch HDR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/program-icons/luminance-hdr</normaloff>:/program-icons/luminance-hdr</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of bracketed images:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="toolTip">
        <string>Select the number of bracketed pictures used to create the HDRs</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="toolTip">
        <string>Select the number of bracketed pictures used to create the HDRs</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="groupBoxOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0">
          <item row="0" column="1">
           <widget class="QComboBox" name="formatComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Choose an HDR output file format</string>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Filename prefix:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="profileComboBox">
            <property name="toolTip">
             <string>Choose one of the creation predefined profiles. 
See documentation for more information.</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Profile 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Profile 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Profile 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Profile 4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Profile 5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Profile 6</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Output format:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="prefixLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>hdr_</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="proposedFilenameCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With leaving this checked, filenames for created HDRs will be of the form &lt;span style=&quot; font-style:italic;&quot;&gt;first-last_HdrCreationModel.ext &lt;/span&gt;where first is the file name of 1st braketed image in the set, last is the file name of the last bracketed image in the set and extension is the selected HDR file format extension.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Use Proposed Filename Scheme</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Response:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Weights:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Predefined profile:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="formatSettingsButton">
            <property name="text">
             <string notr="true">...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Model:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelModel">
            <property name="text">
             <string>Debevec</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelWeights">
            <property name="text">
             <string>Triangular</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="labelResponse">
            <property name="text">
             <string>Linear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxAlignment">
         <property name="title">
          <string>Alignment</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0" colspan="2">
             <widget class="QRadioButton" name="aisRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Use Hugin's align_image_stack engine</string>
              </property>
              <property name="text">
               <string>Hugin's align_&amp;image_stack</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QRadioButton" name="MTBRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Use MTB (Median Threshold Bitmap) engine</string>
              </property>
              <property name="text">
               <string>&amp;MTB</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="autoAlignCheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>Auto align the bracketed pictures</string>
              </property>
              <property name="text">
               <string>Auto-align images</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="autoCropCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Auto-crop</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxIO">
     <property name="title">
      <string>Input/Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QLineEdit" name="outputLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="inputLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="selectOutputFolder">
          <property name="toolTip">
           <string>Folder where created HDRs are saved</string>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="selectInputFolder">
          <property name="toolTip">
           <string>Folder where bracketed pictures are located (in alphabetical order)</string>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Input folder:</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Output folder:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxAg">
     <property name="title">
      <string>Anti-ghosting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Artifact-free High Dynamic Range Imaging</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="threshold_horizontalSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Threshold used by auto anti-ghosting algorithm. A lower threshold means a more aggressive processing but that can lower the final dynamic range. Use the &amp;quot;Recompute&amp;quot; button to see the actual patches coverage and adjust the threshold accordingly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>5000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="threshold_doubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Threshold used by auto anti-ghosting algorithm. A lower threshold means a more aggressive processing but that can lower the final dynamic range. Use the &amp;quot;Recompute&amp;quot; button to see the actual patches coverage and adjust the threshold accordingly&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>O. Gallo and others</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="autoAG_checkBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable auto anti-ghosting algorithm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Auto anti-ghosting</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>formatSettingsButton</tabstop>
  <tabstop>profileComboBox</tabstop>
  <tabstop>proposedFilenameCheckBox</tabstop>
  <tabstop>prefixLineEdit</tabstop>
  <tabstop>autoAlignCheckBox</tabstop>
  <tabstop>aisRadioButton</tabstop>
  <tabstop>MTBRadioButton</tabstop>
  <tabstop>autoAG_checkBox</tabstop>
  <tabstop>threshold_horizontalSlider</tabstop>
  <tabstop>threshold_doubleSpinBox</tabstop>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>selectInputFolder</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>selectOutputFolder</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>BatchHDRDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>721</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAG_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshold_horizontalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAG_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>threshold_doubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAG_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAlignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>aisRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAlignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MTBRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoAlignCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoCropCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proposedFilenameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proposedFilenameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>prefixLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
